src/lib/utils.ts

```diff
@@ -1,10 +1,18 @@
-import { BACKEND_BASE_URL } from './env';
+import { getImageSufix } from '../store/imageCache';
+import { BACKEND_BASE_URL, IMAGE_BASE_URL } from './env';
 
 
 export function url(...args: string[]) {
     return BACKEND_BASE_URL + "/" + args.join("/")
 }
 
+export function imageUrl(...args: string[]) {
+    if (args && args[0]) {
+        return IMAGE_BASE_URL + "/" + args.join("/") + "?t=" + getImageSufix();
```

Comment:
Why dont you use `${IMAGE_BASE_URL}/${args.join("/"}?t=${getImageSufix()}`?

Reply:
Its okay. trust me bro